group_vars/nfss.yml.sample

```diff
@@ -44,6 +44,14 @@ dummy:
 ######################
 #ceph_nfs_log_file: "/var/log/ganesha/ganesha.log"
 #ceph_nfs_dynamic_exports: false
+# If set to true then rados is used to store ganesha exports
+# and client sessions information, this is useful if you
+# run multiple nfs-ganesha servers in active/passive mode and
+# want to do failover
+#ceph_nfs_rados_backend: false
+# Name of the rados object used to store a list of the export rados
+# object URLS
+#ceph_nfs_rados_export_index: "ganesha_rados_export_index"
```

Comment:
s/ganesha_rados_export_index/ganesha-export-index/

'ganesha-export-index' is the default value I've set in the ganesha driver patch. Better to match?